index.html

```diff
@@ -198,15 +198,28 @@ <h3>Easy to install</h3>
 
             <p>
               The easiest way to install CuPy is to use pip.
-              CuPy also supports various versions of CUDA and cuDNN.
-              The installer automatically detects installed versions of CUDA and cuDNN in your environment.
+              CuPy provides Wheels (precompiled binary packages) for the recommended environments. These package include cuDNN and NCCL.
+              <a href="https://docs-cupy.chainer.org/en/stable/install.html#install-cupy">See details</a>.
+
+              And, CuPy can be installed from source code.
+              The installer automatically detects installed versions of CUDA, cuDNN and NCCL in your environment.

```

Comment:
Isn't it confusing? I think it's better to clearly say the installation using a wheel doesn't recognize the versions of CUDA and related libraries. Or just replacing "The installer" with "The install script in the source code" could sound clearer.